addons/snailmail/migrations/13.0.0.1/pre-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_copies = {
+    'snailmail_letter': [
+        ('state', None, None),
+    ],
+}
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.copy_columns(env.cr, _column_copies)
```

Comment:
Can you please explain me whey we do pre-migration for state here if there no change in field? I have also seen in other modules' migration the same process. I am new maybe I am missing something.

Reply:
because later the copied column is used in the post-migration script. the 'draft' option has disappeared from state field.